__tests__/app.test.js

```diff
@@ -113,4 +112,56 @@ describe("get /api/reviews/:review_id",() => {
 
     })
 
+})
+
+// get api/reviews/:review_id/comments
+describe('GET /api/reviews/:review_id/comments',() => {
+    test("check it gives the correct status, and correct data when given a review_id with existing comments",() => {
+        return request(app).get('/api/reviews/2/comments')
+        .expect(200)
+        .then(({body: {comments}}) => {
+            expect(comments).toHaveLength(3)
+            comments.forEach((comment) => {
+                expect(comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        review_id: expect.any(Number)
```

Comment:
We know exactly what the review_id should be if we are going to api/reviews/2/comments 😄